README.md

```diff
@@ -88,3 +88,15 @@ For such binaries where test cases are independent, `gtest-parallel` provides
 `--serialize_test_cases` that runs tests within the same test case sequentially.
 While generally not providing as much speedup as fully parallel test execution,
 this permits such binaries to partially benefit from parallel execution.
+
+## XML Output
+
+This script is able to save test results in XML format. Use
+`--dump_xml_test_results=` to enable XML output and specify file name.
+By default standard error and output are saved in XML for failed tests.
+Include `--dump_xml_all_logs` in command line to save output of all logs.
```

Comment:
Is there a downside to always dumping all log output? I'd prefer not adding more options here (and avoid a new flag specifically per format).

Reply:
I think the downside is potentially large XML file and the need to parse it later. Personally, I save all logs just in case I want to take a look at history but I can imagine people disagree that this is needed.